internal/test/wmcb/wmcb_test.go

```diff
@@ -256,17 +238,10 @@ func (vm *wmcbVM) initializeTestConfigureCNIFiles(ovnHostSubnet string) error {
 		return fmt.Errorf("unable to create remote directory %s: %v\n%s", remoteDir, err, output)
 	}
 
-	cniPkgUrl := framework.pkgs[cniPluginPkgName].getUrl()
-	cniUrl, err := url.Parse(cniPkgUrl)
-	if err != nil {
-		return fmt.Errorf("error parsing %s: %v", cniPkgUrl, err)
-	}
-
-	// Download and extract the CNI binaries on the Windows VM
-	err = vm.remoteDownloadExtract(framework.pkgs[cniPluginPkgName], remoteDir+path.Base(cniUrl.Path), winCNIDir)
+	out, err := vm.Run(fmt.Sprintf("Move-Item -Path %s -Destination %s", remoteDir+path.Base("cni"), winCNIDir), true)
```

Comment:
Why do this instead of just copying the files to where we want in the first place?

Reply:
We dont know the destination while transferring the files to Windows VM. So Transferring it to a temp directory and then copying it to required directory.